docs/git-workflow.md

```diff
@@ -0,0 +1,76 @@
+# Git workflow
+
+As a Mineral UI developer, I would like an improved git workflow, so that I have more control over the release process.
```

Comment:
can you say more about what control you're missing in the current release process?